pipeline.py

```
@@ -1193,35 +1906,205 @@ def build_payload(fields: Dict[str, Any], source_path: Path) -> Dict[str, Any]:
         },
         "meta": {
             "extraction_confidence": fields.get("_confidence"),
+            "ocr_engine": fields.get("_engine"),
+            "extraction_status": fields.get("_status", "unknown"),
+            "errors": fields.get("_errors", []),
+            "warnings": fields.get("_warnings", []),
+            "confidence_thresholds": fields.get("_thresholds", {}),
         },
     }
 
+def _validate_file_format(file_path: Path) -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)